### DISCUSSION TITLE
Add option to exclude (skip) certain filterid's

### DISCUSSION THREAD

**Discussion by puterboy** [bemasher/rtlamr#167] [Ideas]
Awesome program!

Sometimes rather than just including a list of filterid's, one may want to alternatively exclude a list of filterid's -- for example when trying to discover an elusive device. Of course, I can pipe the output onto 'grep -v' to create my own filter, but it would be nice to have as a built in option.

Say -filteridskip=: skip messages matching an id in a comma-separated list of ids.

It may similarly be nice to have similar complements for -filtertype and -msgtype

Seems like this would be pretty trivial to code...

Just an idea...
Jeff

**Comment by bemasher** [comment]
Piping through `grep -v` is probably a more appropriate solution for excluding meters.

**Reply by puterboy** [reply]
Makes sense. Thanks!
